workspace_host/interface.js

```diff
@@ -830,7 +806,7 @@ function initSequence(workspace_id, useInitialZip, res) {
         'id': workspace_id,
         'launch_uuid': uuid,
         'local_name': `workspace-${uuid}`,
-        's3_name': `workspace-${workspace_id}`,
+        's3_name': `workspace-${workspace_id}/current`,
```

Comment:
I suppose it's on purpose since we shouldn't have a `launch_uuid` or `launch_port` yet at this point, and we're going through and setting these fields up.

Reply:
ah nevermind, `launch_uuid` hasn't been set by the `set_workspace_launch_uuid` yet